docs/get-started/celeba.md

```diff
@@ -39,7 +39,7 @@ for doc in data:
 
 ## Load the pretrained model
 
-Let's import pretrained ResNet50 as our base model. ResNet50 is implemented in PyTorch, Keras and Paddle. You can choose whatever framework you feel comfortable:
+Let's import a pretrained ResNet50 as our base model. ResNet50 is implemented in PyTorch, Keras and Paddle. You can choose the framework you feel more comfortable with:
```

Comment:
chose framework you are the most comfortable with
or: 
chose whichever framework you are more comfortable with